### ISSUE TITLE
Slight problem parsing Wxxxx Nxxxx addresses

### ISSUE THREAD

**Issue by njvack** [datamade/usaddress#92]
My in-laws' mailing address:

W148 N9748 Tree Tops Dr
Germantown, WI 53022

which parses as:

```
"AddressNumber","StreetName","StreetNamePostType","PlaceName","StateName","ZipCode"
"W148","N9748 Tree Tops","Dr","Germantown","WI","53022"
```

rather than the correct:

```
"AddressNumber","StreetName","StreetNamePostType","PlaceName","StateName","ZipCode"
"W148 N9748","Tree Tops","Dr","Germantown","WI","53022"
```

**Comment by cathydeng** [comment]
we don't have anything like this in our training data - thanks! I'll train the parser on it.

do you happen to know why there are two address numbers? do W & N specify direction?

**Comment by njvack** [comment]
That's my guess, but I don't know for sure (or, in reference to what). Looking at other Germantown addresses, it seems likely and I think you could triangulate ;-)

FWIW, it took Google Maps quite a while to start understanding this as well.

**Comment by cathydeng** [comment]
closed via 0aa38f5e81a12cd74cf4e83c0ab1caa37b16cae8

usaddress 0.4.6 now handles these types of address numbers correctly. thx!
